databaseFunctions.js

```diff
@@ -0,0 +1,72 @@
+const { db, end } = require("./database.js");
+
+// Takes in task name for creation and outputs the id of the created task
+async function addTask(taskName) {
+  try {
+    const query = `INSERT INTO task (name, completed, created_date)
+                        VALUES ($1, false, NOW())
+                        RETURNING id;`;
+
+    const res = await db.one(query, [taskName]);
```

Comment:
See the comment about returning results directly and avoiding unnecessary async/await.